packages/upload/theme/lumo/vaadin-upload-styles.js

```diff
@@ -123,6 +123,7 @@ const uploadFile = css`
   [part='commands'] {
     display: flex;
     align-items: baseline;
+    align-self: flex-start;
```

Comment:
Maybe we should just change it to use `align-items: flex-start` everywhere. Using baseline as a default doesn't seem particularly useful for the elements that the upload file row contains.

Reply:
It occurred to me that the whole row would be better implemented as a css grid... which we could definitely do in V25, and _possibly_ in a minor.